### PR TITLE
Admin - Users listing + view user

### PR DESCRIPTION
**Pull request by yj-soh**
To run:
Do `npm install` and `grunt` while in `/app_server`. (assuming you have both installed)
Then just open up `/app_server/public/index.htm`.
Page available at "All Users" under "Users" from the left navbar.
View user available by selecting "View / Edit User" from the actions list.

Summarised list of additions:
- Add mock data for users.
- Add user listing page, fetching data from the above mock data.
  - Uses the datadisplay component from #26.
  - Includes a model class for user (for frontend).

To-dos:
- Retrieve users' follower count as well (likely on backend?).

### CODE REVIEW

File: app_server/public/js/modules/mockdata/user.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{ "userId": 2, "username": "alice", "platform_type": "facebook", "alias": "queenofhearts", "description": "I'm Mab, queen of ice and darkness", "email": "alice42@wonderland.org", "createdAt": "2011-04-10T20:09:31Z", "updatedAt": "2014-03-03T18:58:10Z" }
```
Comment (sharminelow) [inline app_server/public/js/modules/mockdata/user.json:1]: convention: empty line after last line

File: app_server/public/js/modules/pages/users.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+const Datadisplay = require('../components/datadisplay');
+
+const platformImg = {
+  facebook: '/admin/img/facebook.png'
+};
+
+const Users = module.exports = {
+  controller: function () {
+    let ctrl = this;
+
```
Comment (sharminelow) [inline app_server/public/js/modules/pages/users.js:14]: `Datadisplay` should be in camelcase

File: app_server/public/js/modules/pages/users.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+          return {
+            id: user.id(),
+            name: user.alias(),
+            followers: '42', // TODO: Get user's follower count
+            platform: getPlatform(user.platform()),
+            actions: getActions(user.id())
+          };
+        }
+    );
+
```
Comment (sharminelow) [inline app_server/public/js/modules/pages/users.js:51]: `followers: '42'` do take note of strings/integers in the fields. And we don't have a 'followers' field. We only have subscribers and subscriptions.

File: app_server/public/js/modules/mockdata/user.json (hunk 1 above)
Comment (yj-soh) [inline app_server/public/js/modules/mockdata/user.json:1]: I'll probably keep `json` files without the new line?
I'm fine either way though, should I keep the convention for them with or without?

**Comment by sharminelow** [comment]
Yeap. Good to merge.

**Pull request merged**